### ISSUE TITLE
Little reminder for the subsequent users for the eigen

### ISSUE THREAD

**Issue by chqwer2** [Rubikplayer/flame-fitting#27]
The original command to download eigen in README will download a version that is too new to use.

The version [eigen-3.3.7](https://gitlab.com/libeigen/eigen/-/releases/3.3.7) will works fine.

**Comment by Suvi-dha** [comment]
Thank you. Could you please help me how I could get 3d landmarks from the scan without manual extraction?

**Comment by ypicard** [comment]
Indeed, this fixed it for me too. I was trying with the Eigen `v3.4.0` version and it failed with the error message:
```
This compiler appears to be too old to be supported by Eigen
```
which seems to be coming from this file: https://eigen.tuxfamily.org/dox-devel/Macros_8h_source.html, line 624.

**Comment by phladwig** [comment]
I fixed build issues by changing C++11 to C++17 in sbody/alignment/mesh_distance/setup.py line 14

**Comment by sxdl** [comment]
> I fixed build issues by changing C++11 to C++17 in sbody/alignment/mesh_distance/setup.py line 14

it works. thank you
